src/components/button/style.module.css

```diff
@@ -0,0 +1,21 @@
+.btn_container {
+  border-radius: 6px;
+  background: #283756;
+  padding: 10px;
+  color: #FFF;
+  min-width: 150px;
+  cursor: pointer;
+}
+
```

Comment:
The focus state has a blue border around it onClick. Might need this?

```css
.btn_container:focus {
  outline: 0;
}
```

Reply:
I actually didn't add that intentionally, the focus ring is used to signify focus for users that access the application without a pointer device (e.g mouse).

You can have a look at [this](https://a11yproject.com/posts/never-remove-css-outlines/) and [this](https://medium.com/hackernoon/removing-that-ugly-focus-ring-and-keeping-it-too-6c8727fefcd2) for alternatives but I'll advise against removing the outline as it serves a purpose (especially considering we want to make this accessible for everyone).

That said if the decision is to remove it, I'm fine with removing the outline. Just say the word.

Reply:
Thanks for the clarification, we should leave it as we want this to be accessible.